user_templates/printer.cfg

```diff
@@ -200,6 +200,8 @@
 # [include config/hardware/accelerometers/adxl345_Fysetc_SB_Can_TH.cfg] # For ADXL plugged in Fysetc SB Can TH boards
 
 # [include config/hardware/accelerometers/lis2dw_usb_rp2040_spi1.cfg] # For BTT S2DW V1.0, ... 
+
+# [include config/hardware/accelerometers/eacon_accelerometer.cfg] # For Beacon Rev H make sure Beacon. Do not forget to include the beacon_virtual_probe in the PROBE section!
```

Comment:
Be careful, there is a typo here in the file name: `eacon_accelerometer.cfg`

Reply:
Good catch, will fix this!